components/dbt/sources/job-run-completed/job-run-completed.mjs

```diff
@@ -0,0 +1,28 @@
+import common from "../common/common.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "dbt-job-run-completed",
+  name: "Job Run Completed (Instant)",
+  description: "Emit new event when a job run has completed.",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getEventTypes() {
+      return [
+        "job.run.completed",
```

Comment:
This source should also return runs that failed, via `job.run.errored`.

But there is a workaround: to go check the box manually in the DBT Cloud IU:
![image](https://github.com/PipedreamHQ/pipedream/assets/66686238/d2bd75c2-29d8-40dd-aa87-a5bb5e453e94)

Reply:
Thanks, @mroy-seedbox. I've created an issue to address your recommendation - https://github.com/PipedreamHQ/pipedream/issues/7635

Reply:
Oh, it looks like I was wrong!

> The `job.run.errored` event is a subset of the `job.run.completed` events. If you subscribe to both, **you will receive two notifications** when your job encounters an error.

So just `job.run.completed` is sufficient! 👌

But the issue you created is nice, because it allows the user to choose. 👍